lib/sycamore/sycamore/tests/unit/transforms/test_detr_partitioner.py

```diff
@@ -64,3 +64,25 @@ def test_batched_sequenced(self):
             s, TEST_DIR / "resources/data/pdfs/basic_table.pdf", use_ocr=True, use_cache=False
         )
         assert len(d) == 1
+
+    def test_table_extraction_order(self):
+        s = ArynPDFPartitioner("Aryn/deformable-detr-DocLayNet")
+        d = check_table_extraction(
+            s,
+            TEST_DIR / "../../../../apps/crawler/crawler/http/tests/visit_aryn.pdf",
```

Comment:
minor: Maybe this is a normal thing for mono-repos, but I wonder if it might not be safer to put these pdfs in the sycamore library test directory just to avoid surprises if someone edits crawlers. I don't feel strongly, and I'm not thrilled with making the repo even bigger, so I'll leave it to you.

Reply:
It is cleaner, and since it allows this to be more modular, I'll copy it over to the test directory as well